template.md

```diff
@@ -0,0 +1,305 @@
+#ms.reviewer: [ALIAS]
+#ms.suite: ems
+#ms.tgt_pltfrm:
+#ms.custom:
+
+---
+
+# Metadata and Markdown Template
+
+This core-docs template contains examples of Markdown syntax, as well as guidance on setting the metadata. To get the most of it, you must view both the [raw Markdown](https://raw.githubusercontent.com/dotnet/core-docs/master/template.md) and the [rendered view](https://github.com/dotnet/core-docs/blob/master/template.md) (for instance, the raw Markdown shows the metadata block, while the rendered view does not).
+
+When creating a Markdown file, you should copy this template to a new file, fill out the metadata as specified below, set the H1 heading above to the title of the article, and delete the content. 
+
+
+## Metadata 
+
+The full metadata block is above (in the [raw Markdown](https://raw.githubusercontent.com/dotnet/core-docs/master/template.md)), divided into required fields and optional fields. Some key notes:
+
+- You **must** have a space between the colon (:) and the value for a metadata element.
+- If an optional metadata element does not have a value, comment out the element with a # (do not leave it blank or use "na"); if you are adding a value to an element that was commented out, be sure to remove the #.
+- Colons in a value (for example, a title) break the metadata parser. In their place, use the HTML encoding for a colon of `&#58;` (for example, `"title: Writing .NET Core console apps&#58 An advanced step-by-step guide"`).
+- **title**: This title will appear in search engine results. The title should end with a pipe (|) followed by the name of the service (for example, see above). The title doesn't need (and probably shouldn't) be identical to the title in your H1 heading. It should be roughly 65 characters (including | SERVICE NAME)
+- **author**, **manager**, **reviewer**: The author field should contain the **GitHub username** of the author, not their alias.  The "manager" and "reviewer" fields, on the other hand, should contain aliases. ms.reviewer specifies the name of the PM associated with the article or service.
+- **ms.assetid**: This is the GUID of the article from CAPS. When creating a new Markdown file, get a GUID from [https://www.guidgenerator.com](https://www.guidgenerator.com). 
+
+## Basic Markdown, GFM, and special characters
+
+All basic and GitHub-flavored Markdown is supported. For more information on these, see:
+
+- [Baseline Markdown syntax](https://daringfireball.net/projects/markdown/syntax)
+- [GitHub-flavored Markdown (GFM) documentation](https://guides.github.com/features/mastering-markdown)
+
+Markdown uses special characters such as \*, \`, and \# for formatting. If you wish to include one of these characters in your content, you must do one of two things:
+
+- Put a backslash before the special character to "escape" it (for example, \\\* for a \*)
+- Use the [HTML entity code](http://www.ascii.cl/htmlcodes.htm) for the character (for example, \&\#42\; for a &#42;).
```

Comment:
`;` is not a special character, so escaping it means the `\` shows up: `&#42\;`.